specification/loadtestservice/LoadTestService/client.tsp

```diff
@@ -43,7 +44,9 @@ interface TestRunOperations {
     GetTestRun is LoadTestRun.GetTestRun;
     GetTestRunFile is LoadTestRun.GetTestRunFile;
     ListMetricDimensionValues is LoadTestRun.ListMetricDimensionValues;
+    @projectedName("python", "get_metric_definitions")
```

Comment:
Just curious if you know why these names changed?

Reply:
context: https://github.com/Azure/azure-rest-api-specs/pull/24801#discussion_r1263217904. I am not very clear why they use such kind of name. But to avoid breaking, we have to do the rename to keep same as before.